iron-selectable.html

```diff
@@ -132,6 +132,8 @@
     attached: function() {
       this._observer = this._observeItems(this);
       this._contentObserver = this._observeContent(this);
+      this._removeListener(this.activateEvent);
```

Comment:
I think there should be some sort of check here (if we have already have a listener we shouldn't remove and re-add it for no reason). In 90% of cases, when you don't actully manually detach and reattach anything, this doesn't need to happen.